### PR TITLE
Faulty binary fix

### PR DESCRIPTION
**Pull request by Freakqency**
the cdogs binary is a le file but it is placed in mz bins folder and tested to satisfy mz conditions , this is causing build failures

### CODE REVIEW

File: new/db/formats/mz/cdogs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 NAME=MZ: corkami cdogs.exe - open
+BROKEN=1
```
Comment (Maijin) [inline new/db/formats/mz/cdogs]: Remove the BROKEN=1 for this one.

Comment (Freakqency) [reply new/db/formats/mz/cdogs]: done.sorry my bad

File: new/db/formats/mz/cdogs [hunk 2]
```diff
@@ -9,6 +9,7 @@ EOF
 RUN
 
 NAME=MZ: corkami cdogs.exe - entrypoint
+BROKEN=1
 FILE=../bins/mz/cdogs.exe
```
Comment (Maijin) [inline new/db/formats/mz/cdogs]: You forgot to change the path

File: new/db/formats/mz/cdogs [hunk 3]
```diff
@@ -19,6 +20,7 @@ EOF
 RUN
 
 NAME=MZ: corkami cdogs.exe - pi 1
+BROKEN=1
 FILE=../bins/mz/cdogs.exe
```
Comment (Maijin) [inline new/db/formats/mz/cdogs]: same here

**Pull request merged**